plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/operations/BasicCreateOperationHandler.java

```diff
@@ -44,8 +44,7 @@ public BasicCreateOperationHandler(final List<String> handledElementTypeIds) {
    @SuppressWarnings("unchecked")
    @Override
    protected Class<T> deriveOperationType() {
-      return (Class<T>) (GenericsUtil.getParametrizedType(getClass(), BasicCreateOperationHandler.class))
```

Comment:
This override seems unnecessary now since it exactly the same as the implementation it the parent class (`AbstractCreateOperationHandler`)

Reply:
Done.